doc/classes/StatusIndicator.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="StatusIndicator" inherits="Node" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
+	<brief_description>
+		Application status indicator (aka. notification area icon).
+		[b]Note:[/b] Status indicator is implemented on macOS and Windows.
+	</brief_description>
+	<description>
+	</description>
+	<tutorials>
+	</tutorials>
+	<members>
+		<member name="icon" type="Image" setter="set_icon" getter="get_icon">
+			Status indicator icon.
+		</member>
+		<member name="tooltip" type="String" setter="set_tooltip" getter="get_tooltip" default="&quot;&quot;">
+			Status indicator tooltip.
+		</member>
+		<member name="visible" type="bool" setter="set_visible" getter="is_visible" default="true">
+			If [code]true[/code], the status indicator is visible.
+		</member>
+	</members>
+	<signals>
+		<signal name="pressed">
+			<param index="0" name="mouse_button" type="int" />
+			<param index="1" name="position" type="Vector2i" />
```

Comment:
I don't have a strong opinion either way, but would there be use cases for float precision for the position? Since it seems to be `mouse_get_position()` which is a `Vector2` originally.

Reply:
`mouse_get_position` return `Point2i` and I do not think there are any conditions when mouse position might be fractional (it's in desktop coordinates, and underlying API use ints).